prototypes/styletransfer/model.py

```diff
@@ -116,15 +116,17 @@ def run_style_transfer(
             layer.trainable = False
 
         # Get the style and content feature representations (from our specified intermediate layers)
-        style_features, content_features = self.feature_representations(
+        content_features, style_features = self.feature_representations(
             content_img, style_img
         )
         gram_style_features = [
             gram_matrix(style_feature) for style_feature in style_features
         ]
 
         # Set initial image
-        init_image = tfe.Variable(content_img, dtype=tf.float32)
+        # TODO(glanaro): is it correct to preprocess or we don't really care?
```

Comment:
I think we must, as `init_img`, `content_img` and `style_img` pass through the same NN and we compare the activations of the intermediate layers in the loss function. Pre-processing (ImageNet stats normalization) is of paramount importance.

Reply:
Great, I'll remove the TODO

Reply:
By accident, I once applied custom normalization (on the validation set) instead of using ImageNet stats  (which I was using on the training set) and the end result on the Deep Learning task was disastrous. Training accuracy dropping to 60% from 95% (binary classification). I could not believe it had such a huge impact. The truth is that the network "sees" a completely different image if preprocessed improperly.